docker/local-metadata/docker-compose-postgres.yml

```diff
@@ -130,15 +125,14 @@ services:
       postgresql:
         condition: service_healthy
     networks:
-      local_app_net:
-        ipv4_address: 172.16.239.13
```

Comment:
Can we do the same change in `docker/local-metadata/docker-compose.yml` to have files aligned?

Reply:
Do you mean the network name of docker compose or ip address ?

Reply:
I have commented in the wrong place. I also wanted to ask if we can apply this change to `docker/local-metadata/docker-compose.yml` since that file still has fixed IP addresses in the service networks.

![image](https://user-images.githubusercontent.com/11074908/190342042-c3e6ee9b-66d9-456d-b523-c5512ab23dd8.png)

Reply:
Just fixed this! Thanks.